test/DynamoCoreWpfTests/ConverterTests.cs

```diff
@@ -593,5 +593,28 @@ public void LibraryViewModeToBoolConverterTest()
             result = converter.Convert(SearchViewModel.ViewMode.LibrarySearchView, null, null, null);
             Assert.IsFalse((bool)result);
         }
+
+        [Test]
+        public void FilePathDisplayConverterTest()
+        {
+            var converter = new FilePathDisplayConverter();
+
+            Assert.DoesNotThrow(() => converter.Convert("\\\\psf\\Home\\Desktop\\somedyn.dyn",null,null,null));
```

Comment:
Stylistic nit - I'd before @"..." here, to stop this crazy Windows \\\\\\\\\\\\\\\\\ing